RACK-Ontology/ontology/REQUIREMENTS.sadl

```diff
@@ -10,62 +10,73 @@
  * of the Defense Advanced Research Projects Agency (DARPA).
  */
 
+/************** edit history *****************
+  *
+  * dwa - 11 Nov 2020 - reduced cardinality of several
+  * dwa - 11 Nov 2020 - removed incorrect associations
+  * dwa - 11 Nov 2020 - removed unnecessary subclassing
+  *
+  *********************************************/
+
 uri "http://arcos.rack/REQUIREMENTS" alias Rq.
-import "http://arcos.rack/PROV-S".
+import "http://arcos.rack/HAZARD".
 
 REQUIREMENT
 	(note "Captures (both high- and low-level) properties of a process or artifact that are to be assessed")
 	is a type of ENTITY.
 
-	text (note "Description of the REQUIREMENT") describes REQUIREMENT with values of type string.
+	text (note "Description of the REQUIREMENT") describes REQUIREMENT with a single value of type string.
 	// The "text" string can optionally be subdivided into sub-parts : givenText, ifText, thenText
 	// This corresponds to how rules are defined in SADL and note that in a SADL rule whatever is in
 	// givenText can simply be included in the ifText part.
 	// Note that modulo naming changes this structure also matches the structure
 	// that GrammaTech had which was : Given, When, Then parts.
 	// The structure of requirements in ASSERT subsume this structure of requirements. 
 	givenText (note "optional part that can be used to set certain conditions apart from other conditions in the ifText part") describes REQUIREMENT with values of type string.
+	givenText describes REQUIREMENT with at most 1 value.
 	ifText (note "conditions under which this requirement holds") describes REQUIREMENT with values of type string.
+	ifText describes REQUIREMENT with at most 1 value.
 	thenText (note "specifies what should hold by this requirement if the requirement conditions are met") describes REQUIREMENT with values of type string.
+	thenText describes REQUIREMENT with at least 1 value.
 
 	governs (note "ENTITY(s) that are the subject of the requirement") describes REQUIREMENT with values of type ENTITY.
-	governs is a type of wasDerivedFrom.
+	governs is a type of wasImpactedBy.
 
 	satisfies (note "Parent ENTITY(s) (e.g. REQUIREMENT) that this REQUIREMENT is derived from") describes REQUIREMENT with values of type ENTITY.
-	satisfies is a type of wasDerivedFrom.
-
-	mitigates (note "ENTITY(s) (e.g. HAZARD) that is being mitigated by this REQUIREMENT") describes REQUIREMENT with values of type ENTITY.
-	mitigates is a type of wasDerivedFrom.
+	satisfies is a type of wasImpactedBy.
 
-	createdBy (note "ACTIVITY(s) (e.g. REQUIREMENT_DEVELOPMENT) that created this REQUIREMENT") describes REQUIREMENT with values of type ACTIVITY.
-	createdBy is a type of wasGeneratedBy.
+	Rq:mitigates (note "ENTITY(s) (e.g. HAZARD) that is being mitigated by this REQUIREMENT") describes REQUIREMENT with values of type HAZARD.
```

Comment:
Why are we limiting this to HAZARD?  What if we are wanting to mitigate something else like a failure condition or something